CONTRIBUTING

```diff
@@ -0,0 +1,27 @@
+CONTRIBUTIONS and CODE REVIEWS
+
+We love contributions from the community. Please send pull requests and raise
+issues! All code submissions require review. We use GitHub pull requests for
+this purpose.
+
+
+COPYRIGHTS and LICENSE
+
+Except as otherwise noted in individual files, nMigen is licensed under the
+2-clause BSD license, which is contained in the LICENSE.txt file.
+
+All authors retain copyright ownership of their contributions.
+
+Authors should note, that unless explicitly excepted, all contributions are
+licensed under the 2-clause BSD license.
+
+
+CONTRIBUTORS
+
+Major contributors to nMigen are:
```

Comment:
Let's leave this part out; the git history alone is more than sufficient.

Reply:
Done. 

Was just thinking: that if you wanted something like this in future, rather than "major contributors" it might be "Contributors with more than 100 PRs".

Reply:
I'm not sure if PR count really matters. In general I don't like having two classes of contributors for any reason.